boards/nxp/frdm_mcxa346/board.cmake

```diff
@@ -5,7 +5,7 @@
 #
 
 board_runner_args(jlink "--device=MCXA346")
-board_runner_args(linkserver "--device=MCXA346:FRDM-MCXA346")
+board_runner_args(linkserver "--device=MCXA166:FRDM-MCXA166")
```

Comment:
I find on latest LinkServer there is "MCXA346:FRDM-MCXA346", but no "MCXA266:FRDM-MCXA266", and also no "MCXA276:FRDM-MCXA276"